internal/graph/graph_test.go

```diff
@@ -661,77 +313,15 @@ func TestConnectedObjectGraph_RelationshipIngresss(t *testing.T) {
 				},
 			},
 		},
-		{
```

Comment:
Removed this test because it's an invalid test. I have a follow up PR that introduces this test back in with a proper model, but our `RelationshipIngresses` needs some work since it has some bugs in the test that replaces this.

Will follow up with this immediately after this PR.